### ISSUE TITLE
No mouse cursor ingame or in file selector

### ISSUE THREAD

**Issue by clebercasali** [ChristopherHX/mcpelauncher-manifest#150]
I can't see the mouse cursor while the game is running.
Not sure if this is intentional, because it looks like the Android version of the game doesn't support mouse.
But it's specially hard to use the "import" skin feature, because I have to navigate the file selector only using the keyboard, with the game window out of focus, and this usually crashes the game.

**Comment by ChristopherHX** [comment]
More Information needed:
This isn't intentional, selecting files only with keyboard is a mess.
For me the mouse isn't hiding while importing skins.

Which OS? (Linux, macOS)
Which Launcher variant? (AppImage, Flatpak, dmg)
Which Launcher version? (fork, ng-v0.1p4)
Which Minecraft Version? (1.16.40.*)

__If you are on Linux__, I also need this
Which Desktop Environment? (gnome, kde, etc.)
Gnome works for me
Which Distro? (ubuntu, archlinux, etc.)
ubuntu 20.04 works for me

**Comment by ChristopherHX** [comment]
> with the game window out of focus, and this usually crashes the game

It doesn't crash there for me either, if the game window go out of focus.

I tested in the past changing focus and the virtual desktop 20+ times while in game, never crashed on my system.

**Comment by clebercasali** [comment]
Hey!

> Which OS? (Linux, macOS)
Linux

> Which Launcher variant? (AppImage, Flatpak, dmg)
Flatpak

> Which Launcher version? (fork, ng-v0.1p4)
I think it's fork, but I'm not sure. How can I confirm?

> Which Minecraft Version? (1.16.40.*)
1.16.40.02
 
> Which Desktop Environment? (gnome, kde, etc.)
LXQt

> Which Distro? (ubuntu, archlinux, etc.)
Lubuntu 20.04

**Comment by ChristopherHX** [comment]
Ok, I will try to reproduce this.
Never tested it against lxqt

**Comment by ChristopherHX** [comment]
The Launcher versions isn't needed as its the flatpak version

**Comment by clebercasali** [comment]
I think I have figured it out.
It only happens when the gamepad is connected.

**Comment by ChristopherHX** [comment]
Oh I get your problem.
The game tells the lancher to hide the cursor while it gets gamepad input.
Yes this happens on all variants of the launcher.
Not quit shure what to do about that.

**Comment by clebercasali** [comment]
Got it.
Could it suspend gamepad input and release the mouse when it moves?
Or alternate between gamepad/mouse by trapping some key or key combination?

**Comment by ChristopherHX** [comment]
Yes that could be implemented and would improve usablility of the Launcher.

For the time beeing, if you want to use mouse while a gamepad is connected run this before open the launcher
In flatpak you can ignore gamepads with
`sudo flatpak override --nodevice=all io.mrarm.mcpelauncher`
revert with
`sudo flatpak override --reset io.mrarm.mcpelauncher`

**Comment by clebercasali** [comment]
Great! Thank you!
